packages/react-devtools-shared/src/devtools/views/Components/InspectedElementPropsTree.js

```diff
@@ -45,7 +48,8 @@ export default function InspectedElementPropsTree({
 
   const canDeletePaths =
     type === ElementTypeClass || canEditFunctionPropsDeletePaths;
-  const canEditValues = type === ElementTypeClass || canEditFunctionProps;
+  const canEditValuesProp = type === ElementTypeClass || canEditFunctionProps;
```

Comment:
nit: I don't think we need a separate `canEditValuesProp`. (It will never be different from `canEditValues`).